tests/aws/services/cloudformation/api/test_changesets.py

```diff
@@ -431,6 +431,50 @@ def test_delete_change_set_exception(snapshot, aws_client):
     snapshot.match("e2", e2)
 
 
+@markers.aws.validated
+def test_create_delete_create(aws_client, cleanups, deploy_cfn_template):
+    # create stack
```

Comment:
nit: The comment here seems a bit lacking in context

e.g. 
```suggestion
    """ test the re-use of a changeset name with a re-used stack name """
```